index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Elson Otake</title>
+</head>
+<body>
+  <header>
+    <ul id="header_list">
+      <li id="company">Origami Inc</li>
+      <li id="menu">
+          <span class="material-symbols-rounded" id="hamburger">menu</span>
+      </li>
+    </ul>
+  </header>
+  <div id="presentation">
+    <div id="pres">
+      <div id="name">Elson Otake</div>
+      <div id="position">Software Developer</div>
+      <div id="descr">I can help you build a product , feature or website<br>Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don't hesitate to contact me.</div>
+    </div>
```

Comment:
Kindly add style to your intro like this so it looks exactly as the template you picked on figma For example you can try this:
#descr {
    font-size: 20px
    font-weight: 400  
    color; #c8c8d4
    text-align: center
   margin: 10px 0
  width: auto
}
Try this and you will see how wonderful your page will look.